pynecone/utils.py

```diff
@@ -1019,10 +1019,18 @@ def format_state(value: Any) -> Dict:
 
     # Convert pandas dataframes to JSON.
     if is_dataframe(type(value)):
-        return {
-            "columns": value.columns.tolist(),
-            "data": value.values.tolist(),
-        }
+        try:
```

Comment:
Maybe instead of doing a try/except, we can add a similar function called `is_polars_dataframe`. The only catch is we don't want `polars` as a dependency to pynecone, so we need to do something similar where we check the `__name__` of the value.

Reply:
Hey, sorry for the late reply.
I've added an is_polars_dataframe function that checks the str of the type object for 'polars'.